auto-cancel.md

```diff
@@ -0,0 +1,164 @@
+The *auto-cancel* strategy defines the behavior of pipelines when they enter the
+queue for execution.
+
+By default, all pipelines will be queued and executed on a first-come, first-served
+basis.
+
+Often this is not the optimal solution, especially during development.
+
+We all head situations where you push some changes, only to realize that you have
+missed something small, so you push a new revision immediately and then find out
+another mistake...
+
+Before *auto-cancel* strategies the only way to get immediate feedback on the
+latest revision was to manually stop all pipelines from obsolete commits.
+
+Now if you set up an auto-cancel strategy this will be done automatically.
+That way you will get faster feedback on revisions that matter while skipping
+all the intermediate ones.
```

Comment:
The temporal structure of `Before *auto-cancel* strategies` and `Now if you set up an auto-cancel` is more appropriate for blogs where you are announcing a new feature.

For documentation, it's better to keep the document time-neutral. Example would be:

Without auto-cancel...

If you set up auto-cancel...